x-pack/test/reporting/functional/reporting.js

```diff
@@ -106,20 +106,21 @@ export default function ({ getService, getPageObjects }) {
           await PageObjects.reporting.checkUsePrintLayout();
           await PageObjects.reporting.clickGenerateReportButton();
           await PageObjects.reporting.clickDownloadReportButton(60000);
+          PageObjects.reporting.clearToastNotifications();
 
           const url = await PageObjects.reporting.getUrlOfTab(1);
           await PageObjects.reporting.closeTab(1);
           const reportData = await PageObjects.reporting.getRawPdfReportData(url);
           const reportFileName = 'dashboard_print';
           const sessionReportPath = await writeSessionReport(reportFileName, reportData);
-          const diffCount = await checkIfPdfsMatch(
+          const percentSimilar = await checkIfPdfsMatch(
             sessionReportPath,
             getBaselineReportPath(reportFileName),
             config.get('screenshots.directory'),
             log
           );
           // After expected OS differences, the diff count came to be around 128k
```

Comment:
Do you remember what the difference came out to be in % so the comments can be adjusted?

Reply:
It was something like .00198